action.yml

```
@@ -4,8 +4,18 @@ inputs:
   pattern:
     required: false
     description: >
```

I think you might need to change this to:

```suggestion
    description: |
```